src/linear_algebra/tensor_product.lean

```diff
@@ -580,6 +580,14 @@ def lift.equiv : (M →ₗ N →ₗ P) ≃ₗ (M ⊗ N →ₗ P) :=
   right_inv := λ f, ext $ λ m n, lift.tmul _ _,
   .. uncurry R M N P }
 
+@[simp] lemma lift.equiv_apply (f : M →ₗ[R] N →ₗ[R] P) (m : M) (n : N) :
+  lift.equiv R M N P f (m ⊗ₜ n) = f m n :=
+by simp only [lift.equiv, uncurry_apply, linear_equiv.coe_mk, linear_map.to_fun_eq_coe]
```

Comment:
I think directly unfolding is probably sensible here:
```suggestion
uncurry_apply f m n
```

Reply:
Agreed, definitely better.